package.json

```diff
@@ -4,9 +4,6 @@
   "description": "Browser compatibility data provided by MDN Web Docs",
   "main": "index.js",
   "types": "index.d.ts",
-  "dependencies": {
```

Comment:
`package-lock.json` also needs to be updated.

Reply:
Yes, I can do this! Doing so may also upgrade the [lockfileVersion](https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json#lockfileversion). Is that all right? I may not be able to avoid this with `npm`.

Sorry for all these "thing a causes thing b" complications. I was attempting to touch as little as possible. 😄

Reply:
I have a version of `npm` that doesn't cause any other changes to `package-lock.json`, the resulting diff is actually trivial if you want to apply it manually:
```diff
--- a/package-lock.json
+++ b/package-lock.json
@@ -466,11 +466,6 @@
       "integrity": "sha1-G2HAViGQqN/2rjuyzwIAyhMLhtQ=",
       "dev": true
     },
-    "extend": {
-      "version": "3.0.2",
-      "resolved": "https://registry.npmjs.org/extend/-/extend-3.0.2.tgz",
-      "integrity": "sha512-fjquC59cD7CyW6urNXK0FBufkZcoiGG80wTuPujX590cB5Ttln20E2UB4S/WARVqhXffZl2LNgS+gQdPIIim/g=="
-    },
     "fast-deep-equal": {
       "version": "3.1.3",
       "resolved": "https://registry.npmjs.org/fast-deep-equal/-/fast-deep-equal-3.1.3.tgz",
```

Reply:
🙏 Thank you.

Done in https://github.com/mdn/browser-compat-data/pull/9821/commits/a00eb8a8f64d57e6eea8b4c96380056b0471f55e